saleor/checkout/calculations.py

```diff
@@ -361,7 +361,7 @@ def _fetch_checkout_prices_if_expired(
     force_update: bool = False,
     database_connection_name: str = settings.DATABASE_CONNECTION_DEFAULT_NAME,
     pregenerated_subscription_payloads: dict | None = None,
-) -> tuple["CheckoutInfo", list["CheckoutLineInfo"]]:
+) -> tuple["CheckoutInfo", list["CheckoutLineInfo"], bool]:
```

Comment:
Could you provide more information about the returned bool variable in the function description?